addons/overpressure/initSettings.inc.sqf

```diff
@@ -1,7 +1,15 @@
 private _category = [LELSTRING(common,categoryUncategorized), LLSTRING(DisplayName)];
 
 [
-    QGVAR(distanceCoefficient), "SLIDER",
+    QGVAR(backblastDistanceCoefficient), "SLIDER",
+    [ELSTRING(statBackblastRange,Overpressure), LSTRING(distanceCoefficient_toolTip)],
+    _category,
+    [-1, 10, 1, 1],
```

Comment:
I think a lower limit of 0 is fine?
0 for disabled is just as good as -1, I'm not sure why we let it go lower


```suggestion
    [0, 10, 1, 1],
```